### ISSUE TITLE
gallery-build not checking for image issues

### ISSUE THREAD

**Issue by addohm** [haltakov/simple-photo-gallery#125]
Heres the error when building
```
Building the Simple Photo Gallery...
Generating thumbnails...
Something went wrong while generating the thumbnails: image file is truncated (6 bytes not processed)
```
Heres the solution on SO.

https://stackoverflow.com/a/23575424/7806300

**Comment by kroesche** [comment]
Do you have a sample image that shows the problem, or a command line that will create one? I would like to be able to reproduce to issue to confirm a fix. You can put it on a temporary drop somewhere so i can pick it up? Thanks.

**Comment by kroesche** [comment]
I found an image i can use to reproduce the problem. Verifying the fix.

**Comment by kroesche** [comment]
Addressed in release 1.5.6
